.github/workflows/push-build-downstream.yaml

```diff
@@ -0,0 +1,59 @@
+name: Push Image Build Downstream
+on:
+  push:
+    branches:
+      - master
+      - release-*
+    tags:
+      - v*
+  pull_request:
+    branches:
+      - master
+
+defaults:
+  run:
+    # reference: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell
+    shell: bash --noprofile --norc -eo pipefail -x {0}
+
+permissions:
+  contents: read
+
+jobs:
+  push-image-to-container-registry:
+    runs-on: ubuntu-20.04
+    if: github.repository == 'red-hat-storage/rook'
+    steps:
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - uses: actions/setup-go@v5
+        with:
+          go-version: "1.21"
+
+        # docker/setup-qemu action installs QEMU static binaries, which are used to run builders for architectures other than the host.
+      - name: set up QEMU
+        uses: docker/setup-qemu-action@master
+        with:
+          platforms: all
+
+      - name: log in to container registry
+        uses: docker/login-action@v3
+        with:
+          registry: quay.io
+          username: paarora+rookds
```

Comment:
Need to use repository secret variables here. We actually already have these secrets defined in the downstream rook fork. To test the PR you should be able to define these secrets in your own fork as well.
```
QUAY_OCS_DEV_ROBOT_USER
QUAY_OCS_DEV_ROBOT_PASSWORD
```

Reply:
but these passwords are for which repo?

Reply:
It should be for https://quay.io/repository/ocs-dev/rook-ceph